CONTRIBUTING.md

```diff
@@ -19,6 +19,29 @@ reported the issue. Please try to include as much information as you can. Detail
 * Any modifications you've made relevant to the bug
 * Anything unusual about your environment or deployment
 
+## Development Environment
+
+1. You will need Python 2 >= 2.7 or Python 3 >= 3.4. Verify which version you have by running `which python`. 
+
+    > If you don't have it installed, download it [here](https://www.python.org/downloads/). When you do this, `pip` should be installed automatically.
+
+1. Install `cfn-lint` from source by cloning the repository from Github and doing a developer install:
+
+    ```bash
+    git clone https://github.com/aws-cloudformation/cfn-python-lint.git
```

Comment:
Think we should encourage contributors to fork instead of cloning a repository they won't be able to push to

Reply:
Nice catch, again :)